openCVLibrary24133/src/main/AndroidManifest.xml

```diff
@@ -4,5 +4,5 @@
       android:versionCode="24133"
       android:versionName="2.4.13.3">
 
-    <uses-sdk android:minSdkVersion="8" />
```

Comment:
Why was this removed?

Reply:
Actually there were some build errors so had to remove this. I'll try doing it again.